continuationsPlugin/src/test/scala/continuations/ContinuationsPluginSuite.scala

```diff
@@ -429,6 +429,52 @@ class ContinuationsPluginSuite extends FunSuite, CompilerFixtures {
       }
   }
 
+  compilerContextWithContinuationsPlugin.test(
+    "it should convert simple suspended def with a named implicit Suspend") {
+    implicit givenContext =>
```

Comment:
```suggestion
    case given Context =>
```

Reply:
done in [4b8f514](https://github.com/47deg/TBD/pull/58/commits/4b8f51434dce80fdbe7b7d9a776119e8d333bf68)